### PR TITLE
Update German translation

### PR DESCRIPTION
**Pull request by Wuzzy2**
100% coverage in PO file, but that does't mean the modpack is translated 100%. Some strings are apparently untranslatable.

### CODE REVIEW

File: homedecor_i18n/locale/de.po [hunk 1]
```diff
@@ -5,1693 +5,1551 @@
...
 
 #: ../building_blocks/init.lua
 msgid "Fake Grass"
-msgstr ""
+msgstr "Falsches Gras"
 
 #: ../building_blocks/init.lua
 msgid "Small bundle of sticks"
-msgstr ""
+msgstr "Kleins Bündel Stöcke"
```
Comment (Thomas--S) [inline homedecor_i18n/locale/de.po]: *Klein<b>e</b>s

File: homedecor_i18n/locale/de.po [hunk 2]
```diff
@@ -5,1693 +5,1551 @@
...
 #: ../itemframes/init.lua
 msgid "Item frame"
-msgstr ""
+msgstr "Ausstellrahmen"
 
 #: ../itemframes/init.lua
-#, fuzzy
 msgid "Item frame (owned by @1)"
-msgstr "%s (gehoert %s)"
+msgstr "Gegenstandsrahmen (gehört @1)"
```
Comment (Thomas--S) [inline homedecor_i18n/locale/de.po]: Why not making this consistent with https://github.com/minetest-mods/homedecor_modpack/pull/378/files#diff-81b5815e538f308631812e0e1e28567cR241?

**Comment by Wuzzy2** [comment]
Both have been fixed!

**Comment by VanessaE** [comment]
Needs rebased.

**Comment by Wuzzy2** [comment]
OMG. There are now conflicts everywhere …
This is going to be !!!FUN!!!

**Comment by Wuzzy2** [comment]
Should work now again.

For future, can you please try to put Homedecor on Weblate?
A few mods are already there:
https://hosted.weblate.org/projects/minetest/minetest/de/

This whole PR thing is quite painful and time-consuming.

**Comment by VanessaE** [comment]
I might consider it at some point.

**Pull request merged**